assets/src/domain/eventEditor/services/context/EntityListContext/TicketsListContext.tsx

```diff
@@ -1,14 +1,32 @@
-import React, { createContext } from 'react';
+import React, { createContext, useMemo } from 'react';
 
-import { EntityListContextProps, ContextProviderProps } from '../types';
+import type { EntityListContextProps, ContextProviderProps } from '../types';
 import { TicketsFilterStateManager, useTicketsListFilterStateManager } from '../../filterState';
+import { useFilteredEntities } from '@appLayout/entityList';
+import { domain, ticketsList } from '@edtrServices/constants';
+import { useTickets } from '@edtrServices/apollo/queries';
+import type { Ticket } from '@edtrServices/apollo/types';
+import { getCacheIds } from '@sharedServices/predicates';
 
-export type TicketsListContextProps = EntityListContextProps<TicketsFilterStateManager>;
+export type TicketsListContextProps = EntityListContextProps<TicketsFilterStateManager, Ticket>;
 
 export const TicketsListContext = createContext<TicketsListContextProps>(null);
 
 export const TicketsListProvider: React.FC<ContextProviderProps> = ({ children }) => {
+	const datetimes = useTickets();
 	const filters = useTicketsListFilterStateManager();
-	const value = { filters };
+
+	const filtersStr = JSON.stringify(filters);
+	// memoize filter state
+	const filterState = useMemo(() => filters, [filtersStr]);
+
+	const filteredEntitiesList = useFilteredEntities(domain, ticketsList, datetimes, filterState);
+
+	const cacheIds = getCacheIds(filteredEntitiesList).join(':');
+	// memoize filtered entities
+	const filteredEntities = useMemo(() => filteredEntitiesList, [cacheIds]);
+
+	const value: TicketsListContextProps = { filterState, filteredEntities };
+
```

Comment:
this piece looks like that one from the dates list, is it worth of extracting this?

Reply:
It does look like that, may be we can extract it later.